server/src/main/java/org/elasticsearch/script/ScriptEngine.java

```diff
@@ -45,4 +46,9 @@
 
     @Override
     default void close() throws IOException {}
+
+    /**
+     * Script contexts supported by this engine.
+     */
+    List<ScriptContext<?>> getSupportedContexts();
```

Comment:
Why a List when we have the info in Set form already?

Reply:
Changed.